src/hooks/useDisputes.js

```diff
@@ -11,30 +11,43 @@ import { convertToString } from '../types/dispute-status-types'
 
 export default function useDisputes() {
   const courtConfig = useCourtConfig()
-  const { disputes } = useDisputesSubscription()
-  const now = useNow()
+  const { disputes, fetching, error } = useDisputesSubscription()
+
+  const now = useNow() // TODO: use court clock
+
+  const disputesPhases = useMemo(() => {
+    if (!disputes) {
+      return null
+    }
+
+    return disputes.map(d => getPhaseAndTransition(d, courtConfig, now))
+  }, [courtConfig, disputes, now])
 
-  const disputesPhases = disputes.map(d =>
-    getPhaseAndTransition(d, courtConfig, now)
-  )
   const disputesPhasesKey = disputesPhases
-    .map(v => convertToString(v[Object.keys(v)[0]]))
+    ? disputesPhases.map(v => convertToString(Object.values(v)[0])).join('')
+    : null
+
+  return useMemo(() => {
+    if (error) {
+      return { error }
+    }
 
-    .join('')
+    if (fetching) {
+      return { fetching }
+    }
 
-  return [
-    useMemo(() => {
-      return disputes.map((dispute, i) => ({
+    return {
+      disputes: disputes.map((dispute, i) => ({
         ...dispute,
         ...disputesPhases[i],
-      }))
-    }, [disputesPhases, disputes, disputesPhasesKey]), // eslint-disable-line react-hooks/exhaustive-deps
-  ]
+      })),
+    }
+  }, [disputesPhases, disputes, disputesPhasesKey, error]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
In cases where we do this, it might be nice to leave a comment explaining why we're not interested in a dependency.